src/dotnet/Fable.Compiler/Fable.Compiler.fsproj

```diff
@@ -40,7 +40,7 @@
       <PackagePath>prefercliruntime</PackagePath>
       <Pack>true</Pack>
     </Content>
-    <Content Include="..\..\..\build\fable-core\" PackagePath="fable-core\" />
+    <Content Include="../../../build/fable-core" PackagePath="fable-core" />
```

Comment:
I remember in previous dotnet SDK version there was a problem where MSBuild wouldn't include subdirectories if you didn't add `\` at the end of the path: https://github.com/NuGet/Home/issues/4795

Apparently it's fixed, but we need to test that :)

Reply:
This one actually was motivated by not being able to open the project in VS2017, it was complaining about the slash at the end.

Reply:
I had to [add back the slash at the end](https://github.com/fable-compiler/Fable/commit/2c26046df98e7cb06c5132a4b52f5127ad15028b#diff-1715577b9f0a4893c7db757910fc247aR43) for the `Include` attribute, without it I was getting the following error when creating the package:

```
  Fable.Compiler -> /Users/alfonsogarciacaronunez/dev/fable/src/dotnet/Fable.Compiler/bin/Release/netcoreapp2.0/dotnet-fable.dll
/usr/local/share/dotnet/sdk/2.1.200/Sdks/NuGet.Build.Tasks.Pack/build/NuGet.Build.Tasks.Pack.targets(198,5): error NU5019: File not found: '/Users/alfonsogarciacaronunez/dev/fable/build/fable-core'. [/Users/alfonsogarciacaronunez/dev/fable/src/dotnet/Fable.Compiler/Fable.Compiler.fsproj]
```

Reply:
I guess there is some disconnect between VS2017 and MSBuild here, perhaps you need to reopen your issue above, if it's still an issue.